modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/DeliveryResponsibility.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.andes.kernel;
+
+
+import org.wso2.andes.subscription.LocalSubscription;
+
+/**
+ * This interface have methods to handle the delivery request
+ */
+public interface DeliveryResponsibility {
```

Comment:
Please rephrase the comment. look at JDK class comments to figure out conventional prose and grammatical structure.